src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html

```diff
@@ -1,102 +1,133 @@
-<div *ngIf="!enrolledCourse" class="rightSideSectionCards ui card sb-course-batch-card">
-    <div class="header batchCardAlign p-16">
-        <span class="header headerBatchCard">{{resourceService?.frmelmnts?.lbl?.batches}}</span>
-        <a class="opacity-1 ml-auto">
-            <sui-select (selectedOptionChange)="getAllBatchDetails()" class="batchCardDropDown"
-                [(ngModel)]="batchStatus" [options]="statusOptions" valueField="value" labelField="name"
-                #batchStatusSelect>
-                <sui-select-option *ngFor="let option of batchStatusSelect?.filteredOptions" [value]="option">
-                </sui-select-option>
-            </sui-select>
-        </a>
+
```

Comment:
All the events which has the course and batch inclusion should have a relationship with cdata in the event